.github/workflows/stale.yml

```diff
@@ -16,16 +16,14 @@ jobs:
       - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          stale-issue-message: "This issue is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 10 days."
-          stale-pr-message: "This PR is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 10 days."
-          close-issue-message: "This issue was closed because it has been stalled for 10 days with no activity."
-          close-pr-message: "This PR was closed because it has been stalled for 10 days with no activity."
+          stale-issue-message: "This issue is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 14 days."
+          stale-pr-message: "This PR is stale because it has been open 14 days with no activity. Remove stale label or comment or this will be closed in 14 days."
+          close-issue-message: "This issue was closed because it has been stalled for 14 days with no activity."
+          close-pr-message: "This PR was closed because it has been stalled for 14 days with no activity."
           stale-issue-label: "stale"
           stale-pr-label: "stale"
-          exempt-all-assignees: true
-          exempt-all-milestones: true
+          exempt-issue-milestones: true
           exempt-issue-labels: "backlog, backlog candidate"
```

Comment:
do we want to keep `backlog candidate` as well? @mpstefan ?

Reply:
Backlog candidate...? Must have been before my time. I don't use the label myself.

Reply:
it's in our https://github.com/nginxinc/nginx-gateway-fabric/blob/main/ISSUE_LIFECYCLE.md